web/ui/react-app/src/pages/alerts/AlertContents.tsx

```diff
@@ -63,6 +64,13 @@ function GroupContent(showAnnotations: boolean) {
 }
 
 const AlertsContent: FC<AlertsProps> = ({ groups = [], statsCount }) => {
+  useEffect(() => {
```

Comment:
As far as I can see, the default value is only set once in this component, so instead of `useEffect` + `useState`, would it be better to use `useMemo` here? That should also cause fewer renders than first processing the `useEffect` and then re-rendering due to the `setDefaultValue()` call:

```typescript
const defaultValue = useMemo(() => ..., [])
```

(similar in the other pages)

Reply:
ready! ;)